win32/pe-parser.lua

```diff
@@ -5,7 +5,7 @@
 -- case of 64 bit fields (bit/flag fields). Pointer arithmetic is still done numerically, so for
 -- very large files this could lead to undefined results. Use with care!
 --
--- Version 0.4, [copyright (c) 2013-2015 Thijs Schreijer](http://www.thijsschreijer.nl)
+-- Version 0.5, [copyright (c) 2013-2018 Thijs Schreijer](http://www.thijsschreijer.nl)
```

Comment:
is this pe-parser 0.6? comment still says 0.5

Reply:
I took it from the 0.6 tag as is: https://github.com/Tieske/pe-parser/blob/version_0.6/src/pe-parser.lua#L8

Reply:
thx @hishamhm fixed those in `master` of `pe-parser`. 

Since there are no code changes, I didn't push a new release. So you can either pick up current master, or stay with this code as it is.